winpr/include/winpr/spec.h

```diff
@@ -965,6 +965,7 @@ extern "C++"
 
 #endif
 
+#if !defined(__MINGW32__)
```

Comment:
Does mingw already define these macros?
Would be no good idea to export all functions for mingw builds

Reply:
yes they are defined in mingw headers.